sdk/python/kfp/components/pipeline_task.py

```diff
@@ -456,6 +463,39 @@ def my_pipeline():
             self._task_spec.dependent_tasks.append(task.name)
         return self
 
+    def set_exit_task(self, exit_task: 'PipelineTask',
+                      **kwargs) -> 'PipelineTask':
+        """Specifies the task to invoke after the current task exits,
+        irrespective of whether the current task completes successfully."""
+        from kfp.components import pipeline_context
+        from kfp.components.tasks_group import ExitHandler
+
+        for _, input_spec in exit_task.component_spec.inputs.items():
+            if not input_spec.default:
```

Comment:
Should this be `.optional`? What happens if the default is falsey (`None` or `''`)?

Reply:
Yes good catch. Thank you